test/server/core/abilities/keyword/Raid.spec.ts

```diff
@@ -80,7 +80,5 @@ describe('Raid keyword', function() {
                 expect(context.p2Base.damage).toBe(4);
             });
         });
-
-        // TODO test that a card that attacked and then is bounced back to hand (i.e. Waylay) doesn't receive a second Raid buff
```

Comment:
I'm OK removing this if we're confident it's not going to happen, but do we have a test to cover this and the similar notes in Restore/Shielded?

Reply:
Mentioned in chat but Chebato did implement these tests, he just missed the comments in some cases